src/centralserver/admin-service/src/test/java/org/niis/xroad/centralserver/restapi/openapi/MembersApiControllerTest.java

```diff
@@ -273,10 +273,10 @@ public class DeleteMember implements WithInOrder {
 
         @Test
         @DisplayName("Should delete client with given id")
-        void shouldReturnMember() {
+        void  shouldReturnMember() {
```

Comment:
accidental extra space added. also why is it called "shouldReturnMember" when it is testing deleteMember call? And another thought that the test is not very meaningful as it doesn't check how code works but rather whether it is still written how it was written.